components/Create/index.tsx

```diff
@@ -83,6 +84,21 @@ export const Create = ({
   // Placeholder message for getting gas params
   const message = useMemo<Message | null>(() => {
     try {
+      const constructorParams = serializeParams({
+        Signers: [...acceptedSigners],
+        NumApprovalsThreshold: approvals,
+        UnlockDuration: vest,
+        StartEpoch: epoch
+      })
```

Comment:
In the return statement below is this check:
```
// Manually check signer validity to prevent passing invalid addresses to createMultisig.
// This can happen due to multiple rerenders when using setIsValid from InputV2.Address.
!acceptedSigners.some((signer) => !validateAddressString(signer))
```

Which prevented an error when passing an invalid signer address to the `createMultisig` method. Maybe this error doesn't occur with `serializeParams`, but it might be a good idea to double check or maybe put the `serializeParams` behind the check anyway